code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -563,11 +562,19 @@ func (ssc *StorageSmartContract) GetAllocationMinLockHandler(ctx context.Context
 	var minLockDemand state.Balance
 
 	ids := append(req.Blobbers, blobbers...)
-	if len(ids) > req.ParityShards+req.DataShards {
-		ids = ids[:req.ParityShards+req.DataShards]
+	uniqueMap := make(map[string]bool)
```

Comment:
Instead of using `bool` as map value, I suggest to use `struct{}`, also init it with capacity

Reply:
what's the difference between bool and struct{}? I suppose they both take 1 WORD size

Reply:
https://itnext.io/set-in-go-map-bool-and-map-struct-performance-comparison-5315b4b107b

Reply:
@dabasov `struct{}{}` doesn't take up any space, while `bool` does, https://dave.cheney.net/2014/03/25/the-empty-struct. We actually don't use the map value, so we don't need to waste resources on unused field.

Reply:
thanks guys